src/test/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -259,4 +264,52 @@ public void testLock() throws Exception {
 
     this.subject.lock();
   }
+
+  @Test
+  public void testBatch() throws Exception {
```

Comment:
we should add tests for some failure cases as well.

also add cases to test for the imperial header being set

Reply:
added imperial test